deployment/README-django.md

```diff
@@ -17,6 +17,10 @@ Default output format [None]:
 
 You will be prompted to enter your AWS credentials, along with a default region. These credentials will be used to authenticate calls to the AWS API when using Terraform and the AWS CLI.
 
+## Env File
+
+A .env file should exist at the root of the source directory.  That must be filled in with mapbox details before running cibuild so that the Django container can access external resources.
+
```

Comment:
We should mention that the `.env` file needs to be in the S3 project settings bucket like the `.tfvars` file.

Reply:
Added clarity around that.